### ISSUE TITLE
PyLNP 0.9.5.a insists on downloading df from Bay12 when trying to switch tilesets

### ISSUE THREAD

**Issue by miffedmap** [Lazy-Newb-Pack/Lazy-Newb-Pack-Linux#1]
@PeridexisErrant, Is there something new LNP needs to play along with?

**Comment by lethosor** [comment]
There have been some graphics-related issues reported in the thread for PE's pack as well. If it's a PyLNP issue, it would probably be better to report it in the [PyLNP repo](https://bitbucket.org/Pidgeot/python-lnp/) (although that may be a little less convenient since it's hosted on BitBucket).

**Comment by miffedmap** [comment]
it makes sense that PyLNP would check for the presence of df before installing stuff.
Easily spotted:
https://bitbucket.org/Pidgeot/python-lnp/src/49aa9b7f9af4920756162b6675af74e00a046536/core/baselines.py?at=default#cl-13

Looks like it's up to LNPL to slipstream the "baseline df" as they call it in a way that conforms
to the directory structure PyLNP expects. Probably just needs a symlink somewhere.

**Comment by miffedmap** [comment]
`ln -sf $PWD/df_linux LNP/Baselines/df_40_24`

**Comment by lethosor** [comment]
As I understand it, the "baseline" DF exists to provide an ancestor for mod changes, so symlinking it to df_linux would cause PyLNP to think that no mods are installed.
It should be simple to distribute a vanilla raw folder, though. @PeridexisErrant what files are required in the baseline folder? Just the `raw` directory?

**Comment by PeridexisErrant** [comment]
Per [this line](https://bitbucket.org/Pidgeot/python-lnp/src/49aa9b7f9af4920756162b6675af74e00a046536/core/baselines.py?at=default#cl-79), the baselines should include `raw/*`, `data/art/*`, `data/init/*`, and `data/speech/*`.

These files are then used as the ancestors for diff-based merging, reconstruction, and installation of mods and graphics.  Don't leave anything out!

The auto-download of a missing baseline version will grab the windows small file version; this is fine though as all platform-specific files are discarded.  The windows version is just for the sake of minimal download volume.  Dropping in a zipped windows version (issues raised, will generalise) or any platform DF version unpacked to the correct folder will work - PyLNP will delete the surplus files next time the baseline is used.

TLDR - the files should be bundled in LNPL, or they can be auto-downloaded.
